github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
I will make a new issue for refactoring. But the discussion of what you propose as refactor can be here as it came up in this PR.  Here are all the list of struct names :

```
type GitPRs struct {
 // holds PR information
}

type PRIssues struct {

 // Holds PR Issues returned from Github.  PRs are a special type of Issues.
}

type GitIssues struct {
 //Holds all the Issue information we need in buggy.
}

type Repoissue struct {
 // Holds all our issue information from repos mentioned in repo.toml
}
```
Please advise.